homework/index.js

```diff
@@ -27,21 +30,138 @@
         elem.setAttribute(key, value);
       }
     });
+
     return elem;
   }
 
-  function main(url) {
-    fetchJSON(url, (err, data) => {
-      const root = document.getElementById('root');
-      if (err) {
-        createAndAppend('div', root, { text: err.message, class: 'alert-error' });
-      } else {
-        createAndAppend('pre', root, { text: JSON.stringify(data, null, 2) });
-      }
+  // fetch url contributor
+  function fetchContributorsUrl(contributor) {
+    const cont = document.getElementById('cont');
+    cont.innerHTML = '';
+    fetchJSON(contributor)
+      .then(myContributor => {
+        for (let j = 0; j < myContributor.length; j++) {
+          const nameOfContributor = myContributor[j].login;
+          const numberOfContributor = myContributor[j].contributions;
```

Comment:
Should be `numberOfContributions`
Also the link can be found in `myContributor[j].html_url`, no need to compose it manually.